packages/@lwc/engine-core/src/framework/modules/static-parts.ts

```diff
@@ -105,6 +107,10 @@ export function mountStaticParts(root: Element, vnode: VStatic, renderer: Render
         // Refs must be updated after every render due to refVNodes getting reset before every render
         applyRefs(part, owner);
         patchAttributes(null, part, renderer);
+        // Note, applyStyleScoping will apply the synthetic shadow style token, however, this is already applied at compile time and when the
+        // fragment is assembled in buildParseFragmentFn.
```

Comment:
```suggestion
        // Note, applyStyleScoping will apply the synthetic shadow style token (i.e. `elm.$shadowToken$`), however, the class/attribute
        // is already applied at compile time and when the fragment is assembled in buildParseFragmentFn.
```

This is correct, right?

Reply:
I ended up removing `applyStyleScoping` so we don't need this anymore!